soc/arm/st_stm32/stm32h7/mpu_regions.c

```diff
@@ -0,0 +1,66 @@
+#include <devicetree.h>
+#include <arch/arm/aarch32/cortex_m/mpu/arm_mpu.h>
+
+/* Flash Region Definitions */
```

Comment:
Copied this from `soc/arm/common/cortex_m/arm_mpu_mem_cfg.h`. Can we share this somehow?

Reply:
Isn't it possible to include `soc/arm/common/cortex_m/arm_mpu_mem_cfg.h` directly ?

Reply:
Included it directly. Wasn't sure because it seems only possible using a relative path:

```
#include "../../common/cortex_m/arm_mpu_mem_cfg.h"
```